docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -142,13 +143,25 @@ $ echo $?
 # 1
 ```
 
-When logging in, the `nginxrestarter` user and any groups that do not already exist
-will be created on the host. The `nginxrestarter` user will be added to the `ubuntu`,
-`nginx` and `other` groups as specified in the `host_groups` field.
-
-A file will also be created in `/etc/sudoers.d` with the contents of the
-`host_sudoers` file written with one entry per line, each prefixed with the username
-of the user that has logged in.
+When the user above logs in, the `nginxrestarter` user and any groups that do
+not already exist will be created on the host. The `nginxrestarter` user will be
```

Comment:
```suggestion
not already exist are created on the host. The `nginxrestarter` user is
```

In my experience, present tense is generally preferred (easier to parse and translate).